### ISSUE TITLE
Add option for reverse sort

### ISSUE THREAD

**Issue by pigmonkey** [firecat53/urlscan#90]
I would like to be able to pass urlscan a flag telling it to return the URLs in the reverse order.

For example, given the following file:

    $ cat urls.txt
    https://github.com/firecat53/urlscan
    https://github.com/
    https://www.google.com/

urlscan will currently list the URLs in the same order as were given in the file:

    $ urlscan -c urls.txt
    [1]   < https://github.com/firecat53/urlscan
    [2]   < https://github.com/ 
    [3]   < https://www.google.com/

Most of the time this is the desired behavior, but some of the time I'd rather the URLs be returned like this:

    [1]   < https://www.google.com/
    [2]   < https://github.com/ 
    [3]   < https://github.com/firecat53/urlscan

This is most often useful when using urlscan on terminal output (such as via [tmux-urlscan](https://github.com/fszymanski/tmux-urlscan)). In this case, I'm trying to get to the most recent URL, which is at the bottom of the terminal. Telling urlscan to reverse sort the URLs allows me to get to this most recent URL without me having to navigate the list.

**Comment by firecat53** [comment]
You know that you can get to the bottom of the list with just a `G`? Is that any slower than passing a separate flag?

**Comment by pigmonkey** [comment]
Yes, for my specific use case I would configure tmux-urlscan to always pass this reverse flag, so this would save me two keystrokes (`shift`, `g`) every time I use urlscan.

**Comment by brett** [comment]
It might be nice to have a config option to specify the default sort order, too.

**Comment by firecat53** [comment]
Give the `reverse` branch a try. `-N` does the same thing as `-n` but prints out the URLs in reverse order. It _only_ address URL output to console and does not affect the TUI at all. Is that what you were looking for?

**Comment by pigmonkey** [comment]
That does result in the desired ordering, but I want to use it with `-c`/`--compact`. My use case for this is quickly opening the most recently printed URLs in my terminal via [tmux-urlscan](https://github.com/fszymanski/tmux-urlscan), which is only helpful with urlscan's TUI browser. (I don't need the URL context, though.)

**Comment by firecat53** [comment]
Ah. I misunderstood you, then. Let me try again...

**Comment by firecat53** [comment]
Just FYI...I have this as a one-liner in my tmux.conf that doesn't require a separate plugin:

    bind-key u capture-pane \; save-buffer /tmp/tmux-buffer \; new-window -n urlscan "urlscan -d /tmp/tmux-buffer"

**Comment by firecat53** [comment]
Ok, give the `reverse` branch a try again. `R` will reverse URLs and context both in the TUI and to stdout.

**Comment by pigmonkey** [comment]
`-R` works perfectly. Thanks!
